pkgs/development/tools/lalrpop/default.nix

```diff
@@ -37,10 +37,11 @@ rustPlatform.buildRustPackage rec {
   '';
 
   meta = with lib; {
+    changelog = "https://github.com/lalrpop/lalrpop/blob/${src.rev}/RELEASES.md";
```

Comment:
This is ugly.
Usually the first attribute of meta is description or homepage; the rest is in alphabetical order.